packages/charts/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -828,7 +830,6 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
         graph.push(
           <g
             key={`${index}-dots-${this._uniqueIdForGraph}`}
-            d={area(singleStackedData)!}
```

Comment:
>singleStackedData [](http://example.com/codeflow?start=20&length=17)

why remove this

Reply:
the `d` attribute only works with `<path>` and has no effect on `<g>`